technology/web/src/main/java/com/github/mizool/technology/web/CacheHeaderFilter.java

```diff
@@ -0,0 +1,94 @@
+/**
+ *  Copyright 2017 incub8 Software Labs GmbH
+ *  Copyright 2017 protel Hotelsoftware GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.github.mizool.technology.web;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CacheHeaderFilter implements Filter
+{
+    private static final Pattern STATIC_ELEMENTS_PATH_PATTERN = Pattern.compile(
+        "^/(webjars|\\d+(\\.\\d+)*(-[\\dA-Za-z]+)?)/.+$");
```

Comment:
hmm, the `(-[\\dA-Za-z]+)?)` looks like it was meant to catch `-SNAPSHOT` as well, but fails to do that for some reason I can't easily make out.

as it makes more sense to disable caching of snapshot artifacts, we should simplify the pattern to `^\\/(webjars|\\d+(\\.\\d+)*)\\/`

also, add a link http://regexr.com/3g43r

note that this will remove support for enabling caching for versions like `1.0-beta3`, which is okay as we don't use those.